src/function/maybe_changed_after.rs

```diff
@@ -557,6 +545,33 @@ where
                     accumulated: inputs,
                 }
             }
+
+            QueryOriginRef::Assigned(_) => {
```

Comment:
This change is just to put the more frequent match arm first?

Reply:
Yeah. I find this also helps with readability because I can reason about the common case first